scripts/theia-version.js

```diff
@@ -0,0 +1,52 @@
+/********************************************************************************
+ * Copyright (C) 2019 Ericsson and others.
+ *
```

Comment:
I am not sure if adding our license in this case is the right thing to do. We used the pattern before in the main repo, but it was for MIT-licensed copied code, which allows re-licensing. I do not know that we can do the same with EPL2. I think the safe thing to do is just keep the original license header and do our own changes under that license.

Reply:
Remove Ericsson copyright and kept the TypeFox one.
Uddated the year of the copyright